### PR TITLE
chore: Release 05/23/2022

### PR DESCRIPTION
**Pull request by dbchristopher**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "deduplicate": "yarn-deduplicate -s fewer yarn.lock",
     "playground": "yarn workspace playground develop",
     "gatsby": "yarn workspace www start",
-    "fix": "yarn lint:es --fix",
+    "fix": "yarn lint:es --fix && yarn prettify-fix",
+    "prettify-fix": "yarn prettier --config='.prettierrc.json' packages/* --write",
```
Comment (dbchristopher) [inline package.json:43]: I found myself needing to manually run `yarn prettier` to be able to get the precommit hooks to run. I figured that should probably just be an included step in the `yarn fix` command

File: packages/components/package.json [hunk 2]
```diff
@@ -56,6 +58,15 @@
...
     "overrides": [
+      {
+        "files": [
+          "*.mdx",
+          "*.md"
+        ],
+        "rules": {
+          "import/no-webpack-loader-syntax": "off"
+        }
+      },
```
Comment (dbchristopher) [inline packages/components/package.json:69]: @treshugart I disabled that webpack loader rule for mdx files. It is weird that it didn't get flagged in our source repo, but I can duplicate this change over there for good measure

**Review by treshugart — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (treshugart) [reply package.json:43]: Should this be duplicated in our source repo?

Comment (dbchristopher) [reply package.json:43]: No need! The root package.json already has a `prettify-fix` script

**Pull request merged**